safeeyes/plugins/smartpause/plugin.py

```diff
@@ -194,7 +194,8 @@ def __start_idle_monitor():
                 smart_pause_activated = True
                 idle_start_time = datetime.datetime.now() - datetime.timedelta(seconds=system_idle_time)
                 logging.info('Pause Safe Eyes due to system idle')
-                disable_safeeyes(None, True)
+                info = _('Paused Safe Eyes due to system being idle')
+                disable_safeeyes(info, True)
```

Comment:
Hmm, this still doesn't work for me - now it fails with
`TypeError: SafeEyes.__init__.<locals>.<lambda>() takes 1 positional argument but 2 were given`
I guess the `is_resting` parameter has to be passed along in the lambda in `safeeyes/safeeyes.py:74` as well:
https://github.com/slgobinath/SafeEyes/blob/6ac8b602bc7a8ecd9fc3d0c2718c04a76e406471/safeeyes/safeeyes.py#L74-L75

Reply:
Yeah, looks like it.